### ISSUE TITLE
Talkback is not reading the disabled chapter names in the lessons tab

### ISSUE THREAD

**Issue by KolliAnitha** [oppia/oppia-android#4363]
**Describe the bug**
Talkback is not reading the disabled chapter names in the lessons tab

**To Reproduce**
1. Go to 'All Topics' - Select any topic
2. Go to Lessons tab
3. Expand to view the chapters
4. Enable talkback
5. On swiping next talkback reads only enabled chapter names

**Expected behavior**
Talkback should read the chapter name as disabled so that talkback users can understand that there are other chapters available in the list

https://user-images.githubusercontent.com/9396084/231125633-0d00fd75-6c91-48c1-9f07-aa2d0894435d.mp4

Environment:
Device name: One plus Nord2 5G
Android version : Android 11
App version: 0.7-alpha-6c08d882d3

**Comment by BenHenning** [comment]
Nice find. This one will require some expert guidance, I think. The general recommendation is to not change the default Android/Talkback behaviors in cases when elements are disabled (to ensure consistency between screenreader & non-screenreader users), but disabled buttons and list items sometimes come up as exceptions. We'll need guidance to figure out how to best proceed.

Taking this so that I can consult some folks to determine next steps.

**Comment by BenHenning** [comment]
For searching context, this issue was found in 0.7-alpha (MR4).

**Comment by rt4914** [comment]
This exact issue is definitely not valid anymore because the UI of the app have been updated but the talkback experience needs to fixed for sure.
Have a look at this video for current output:


https://user-images.githubusercontent.com/9396084/231125633-0d00fd75-6c91-48c1-9f07-aa2d0894435d.mp4


@BenHenning We need to decide whether to read disabled options or not for screen-reader users. Once that is decided than only the parts of this issue can be finalised.

(I have updated the issue description too)

**Comment by kkmurerwa** [comment]
Hi @rt4914. The issue of non-descriptive accessibility text on the lessons was fixed on [PR #4958](https://github.com/oppia/oppia-android/pull/4958). I have attached a video below of how it currently works. I would consider this issue as resolved unless there is something else that needs to be done about accessibility on locked lessons.


https://github.com/oppia/oppia-android/assets/18438114/33dd7027-6dfc-446e-9f06-306a4402292e
